src/test/java/io/cdap/plugin/couchbase/source/CouchbaseSourceConfigTest.java

```diff
@@ -45,7 +48,7 @@
                     Schema.Field.of("boolean_field", Schema.nullableOf(Schema.of(Schema.Type.BOOLEAN))),
                     Schema.Field.of("decimal_field", Schema.nullableOf(Schema.decimalOf(10, 4))));
 
-  private static final CouchbaseSourceConfig VALID_CONFIG = CouchbaseSourceConfigBuilder.builder()
+  private static final CouchbaseSource.CouchbaseSourceConfig VALID_CONFIG = CouchbaseSourceConfigBuilder.builder()
```

Comment:
why did you move config class inside plugin class?

Reply:
I did it after creating the `CouchbaseConfig` parent class. Now it seems that config classes are too large to be kept inside plugin classes. Moved them back. Thanks.